### ISSUE TITLE
[Bug]: [BAD_HTTP_STATUS: 403] Potential causes: IP block, streaming URL deciphering failed or music video geo-block

### ISSUE THREAD

**Issue by richierich-coder** [FreeTubeApp/FreeTube#7143]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

any video it shows this error and no video is playing

![Image](https://github.com/user-attachments/assets/4df62ecf-4fa7-4dcc-a359-6fbf1c6d8b34)

### Expected Behavior

play

### Issue Labels

accessibility issue

### FreeTube Version

v0.23.2 Beta

### Operating System Version

windows 11

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.23.1

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Update to v0.23.3 Beta

**Comment by sushinateur** [comment]
Started happening again today.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@sushinateur is it continuous as in you arent able to play any video

**Comment by sushinateur** [comment]
Hi @efb4f5ff-1298-471a-8973-3d47447115dc . It was every video I tried. 
Message was "[BAD_HTTP_STATUS: 403] Potential causes: IP block or streaming URL deciphering failed". 

But as of writing this, it seems to have gone back no normal. Maybe an issue in InnterTube that got fixed by YT.
Sorry for alarming you prematurely.

**Comment by Tetriser** [comment]
Can confirm, it happened again
![Image](https://github.com/user-attachments/assets/56ba105d-2bec-418f-8495-8ec128a43dcf)

**Comment by Hanuko33** [comment]
It's that time of the month again

![Image](https://github.com/user-attachments/assets/01dbf982-6492-426c-974b-dabbc9ad7cee)

**Comment by Kurpaph** [comment]
Guess what's happening again?

![Image](https://github.com/user-attachments/assets/ae713eda-09af-4db2-8784-9936887075f8)
